src/interfaces/pjson.ts

```diff
@@ -149,6 +149,7 @@ export namespace PJSON {
           debounce?: number
           rollout?: number
         }
+        disableNpmLookup: boolean
```

Comment:
Can you make this optional? `disableNpmLookup?: boolean`

Reply:
Of course, my bad! Done